### ISSUE TITLE
Elevation circulating tetracosanoic (C24) and hexacosanoic (C26) acid concentration

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7570]
**Preferred term label:**
Elevation circulating tetracosanoic (C24) and hexacosanoic (C26) acid concentration  (two terms)

**Synonyms**

**Definition (free text, please give PubMed ID)**
Elevated concentration of tetracosanoic acid (a C24 straight-chain saturated fatty acid) in the blood circulation.
Elevated concentration of hexacosanoic acid (a C26 straight-chain saturated fatty acid) in the blood circulation.

**Parent term (use hpo.jax.org/app)**
Abnormal circulating fatty-acid anion concentration
(Fatty acid ... should also be child of monocarboxylic acid)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
614872 
PMID: 17336976  1/1 (each)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
